plugins/CoreHome/vue/src/Alert/Alert.adapter.ts

```diff
@@ -0,0 +1,52 @@
+import { createApp, ref } from 'vue';
+import Alert from './Alert.vue';
+
+interface ActivityIndicatorAdapterScope extends ng.IScope {
+  severity: string;
+}
+
+export default function alertAdapter(): ng.IDirective {
+  return {
+    restrict: 'A',
+    transclude: true,
+    scope: {
+      severity: '@piwikAlert',
+    },
+    template: '<div ng-transclude/>',
+    compile: function alertAdapterCompile() {
+      return {
+        post: function alertAdapterPostLink(
+          scope: ActivityIndicatorAdapterScope,
```

Comment:
Is that scope on purpose?

Reply:
It's a function argument w/ the type, so it's there on purpose. Is there something missing?

Reply:
Oh I see what you mean, the type name is wrong here.